libs/SalesforceSDK/build.gradle.kts

```diff
@@ -30,6 +30,7 @@ dependencies {
     implementation("androidx.core:core-ktx:1.15.0")
     implementation("androidx.activity:activity-compose:1.10.0")
     implementation("androidx.activity:activity-ktx:1.9.3")
+    implementation("androidx.activity:activity-compose:1.9.3")
```

Comment:
NIT:  `androidx.activity:activity-ktx` and `androidx.activity:activity-compose` are [the same dependency](https://developer.android.com/jetpack/androidx/releases/activity#1.9.3).  I'd like to start organizing these together with a variable at the top like we did for `composeVersion` and `livecycleVersion`.

Reply:
Good idea. Done here: https://github.com/forcedotcom/SalesforceMobileSDK-Android/pull/2658/commits/c6bf677ee80e525d7143d84494a733d031a39c4c